packages/vuetify/src/locale/fr.ts

```diff
@@ -6,7 +6,7 @@ export default {
     loadingText: `Chargement de l'élément...`,
   },
   dataTable: {
-    itemsPerPageText: 'Lignes par page:',
+    itemsPerPageText: 'Lignes par page :',
```

Comment:
Could you elaborate on this change please?

Reply:
This is a French typographic rule. Add a non-break space before `:?!«`